app/lib/features/chat_ng/widgets/message_timestamp_widget.dart

```diff
@@ -4,8 +4,13 @@ import 'package:jiffy/jiffy.dart';
 
 class MessageTimestampWidget extends StatelessWidget {
   final int timestamp;
+  final Color? textColor;
```

Comment:
similar to the other style passing cases, I think this should be having the information passed in here as well. We are in the `chat`-area of things, so passing `bool isMe` and have the widget decide on the specific design rather than the outer pass in depending on the same flag seems more logical to me.


At least, if I wanted to change the design of the timestamp, this is were I would look and wanted to change it.